ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -236,6 +239,9 @@ interface {{ ethernet_interface }}
 {%         if ethernet_interfaces[ethernet_interface].isis_network_point_to_point is arista.avd.defined(true) %}
    isis network point-to-point
 {%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].isis_hello_padding is arista.avd.defined(false) %}
```

Comment:
missing documentation, molecule and readme update for this change to `eos_cli_config_gen`